http-gateway/swagger.yaml

```diff
@@ -305,13 +305,14 @@ paths:
         - 'Resources'
       summary: 'Retrieve a resource'
       description: |
-        The plgd cloud builds an up to date [Shadow](https%3A%2F%2Fplgd.dev%2Ffeatures%2Fdevice-shadow%2F) for each published resource. Retrieving it's content using this API returnes an eventually consistent shadow content of the resource. There is a possibility bypass the shadow by setting the query parameter `shadow` to `false.`
+        The plgd cloud builds an up to date [Shadow](https%3A%2F%2Fplgd.dev%2Ffeatures%2Fdevice-shadow%2F) for each published resource. Retrieving it's content using this API returnes an eventually consistent shadow content of the resource. There is a possibility to bypass the shadow by setting the query parameter `shadow` to `false` or by setting set the query parameter interface. In this case, you can set a query parameter validUntil, where you can define the expiration of the command. 
```

Comment:
"or by setting the `interface` query parameter".

"which allows you to set the command expiration."